### PR TITLE
Rack-wide capacity monitoring.

### PR DESCRIPTION
**Pull request by nzoschke**
On boot and every minute after, check the latest ECS Service Events for any sign of trouble. Notify

### CODE REVIEW

**Comment by nzoschke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;yhlDrx73&#x2F;269-web-hook-on-rack-out-of-capacity"> Web hook on rack out of capacity</a>

File: api/models/process.go [hunk 1]
```diff
@@ -67,6 +68,54 @@ func GetServices(app string) ([]*ecs.Service, error) {
 	return dres.Services, nil
 }
```
Comment (csquared) [inline api/models/process.go]: include this note:

//NOTE: the DescribeService Service.Event contains a maximum of 100 of the latest events
// http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Service.html

**Comment by csquared** [comment]
:+1:

**Comment by csquared** [comment]
:+1:

**Pull request merged**